src/NHSD.BuyingCatalogue.Documents.API/NHSD.BuyingCatalogue.Documents.API.csproj

```diff
@@ -25,11 +25,9 @@
     <PackageReference Include="Swashbuckle.AspNetCore" Version="5.0.0" />
   </ItemGroup>
 
-
-
   <ItemGroup>
     <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
-      <_Parameter1>$(AssemblyName).Tests</_Parameter1>
+      <_Parameter1>$(AssemblyName).Unit.Tests</_Parameter1>
```

Comment:
This will need to the changed due to the suggestion above regarding the name of the Unit Test project.